lib/smart_answer_flows/marriage-abroad/outcomes/_link_to_consular_fees.govspeak.erb

```diff
@@ -1,3 +0,0 @@
-<% if calculator.services.any? %>
-  You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for <%= calculator.country_name_lowercase_prefix %>](/government/publications/<%= calculator.ceremony_country %>-consular-fees).
-<% end %>
```

Comment:
Minor: It might be worth mentioning that this inlined partial is still rendered inside a `if calculator.services.any?` statement in the `_services_and_fees` partial, i.e. why there is no need to copy the `if` statement across. In some ways it might've been clearer to remove the `if` statement in this partial as a separate step.